MetadataExtractor/Formats/Xmp/XmpReader.cs

```diff
@@ -130,7 +130,7 @@ public XmpDirectory Extract([NotNull] byte[] xmpBytes)
         /// <remarks>
         /// The extraction is done with Adobe's XMPCore library.
         /// </remarks>
-        public XmpDirectory Extract([NotNull] string xmpString) => Extract(Encoding.UTF8.GetBytes(xmpString));
+        //public XmpDirectory Extract([NotNull] string xmpString) => Extract(Encoding.UTF8.GetBytes(xmpString));
```

Comment:
Again, I debated this as well so left it in there commented-out. As StringValue is used to hold the original XMP bytes from a file, the byte[] overload of this method (`public XmpDirectory Extract([NotNull] byte[] xmpBytes)`) seemed to accomplish the internal requirements. Before this, the bytes were converted to UTF-8 strings when read, and then passed to this function - which turned them back into bytes. A version that assumes a certain encoding could be prone to issues - even from a string supplied by the user.

Correct me if I'm wrong - the XMP spec indicates UTF-8 is preferred. However, other encodings are accepted. Can you know for sure that `string xmpString` is UTF-8? Perhaps if this version accepted an encoding input it might still be safe, something like
`public XmpDirectory Extract([NotNull] string xmpString, Encoding encoder) => Extract(encoder.GetBytes(xmpString));`
I can see why users might call this directly with their own XMP string (probably so XmpCore can parse it for them) but imposing a UTF-8 encoding here doesn't seem like a good idea.